support-models/src/test/scala/com/gu/support/SerialisationTestHelpers.scala

```diff
@@ -30,7 +30,7 @@ trait SerialisationTestHelpers extends Matchers {
       objectChecks: T => Assertion = (_: T) => succeed,
   ): Assertion =
     decodeResult.fold(
-      e => fail(e.getMessage),
+      e => throw new RuntimeException(e),
```

Comment:
"fail" causes a TestFailedException, and scalatest is smart? enough to only print the line of stack trace where the fail call is.  This makes sense if it's in your actual test, but when it's in a helper function it's not enough information.

In conjuction with the -oS option to scalatest (see build.sbt above) this provides enough information to find out where in the actual test it failed.